application/src/main/java/net/corda/v5/application/interop/AliasMemberInfo.java

```diff
@@ -0,0 +1,35 @@
+package net.corda.v5.application.interop;
+
+import net.corda.v5.base.annotations.CordaSerializable;
+import org.jetbrains.annotations.NotNull;
+import java.util.List;
+
+/**
+ * <p>This interface represents the alias member details like x500name, facadeIds etc.
+ *
+ */
+@CordaSerializable
+public interface AliasMemberInfo {
```

Comment:
why is this an interface and not simply a data class ?

Reply:
Initially I thought about it, but when I looked at other corda-api modules all modules have interface for such data. I took the reference of membership and kept it consistent.

Reply:
OK thanks for clarifying